inspirehep/dojson/experiments/fields/bd1xx.py

```diff
@@ -82,32 +83,49 @@ def contact_details(self, key, value):
     }
 
 
-@experiments.over('name_variants', '^419..')
-@utils.for_each_value
-def name_variants(self, key, value):
-    """Variants of the name."""
-    return value.get('a')
-
-
 @experiments.over('description', '^520..')
 @utils.for_each_value
 def description(self, key, value):
     """Description of experiment."""
     return value.get("a")
 
 
-@experiments.over('spokesperson', '^702..')
+@experiments.over('spokepersons', '^702..')
 @utils.for_each_value
-def spokesperson(self, key, value):
-    """Spokesperson of experiment."""
-    return value.get("a")
+def spokespersons(self, key, value):
+    """Spokepersons of the experiment."""
+    def _get_record(i_values):
+        i_value = force_single_element(i_values)
+        if i_value:
+            try:
+                recid = int(i_value.split('-')[1])
+                return get_record_ref(recid, 'authors')
+            except (TypeError, ValueError):
+                return None
+
+    def _is_current(z_values):
+        z_value = force_single_element(z_values)
+        if z_value and isinstance(z_value, six.string_types):
+            return z_value.lower() == 'current'
+
+    record = _get_record(value.get('i'))
+
+    return {
+        'name': force_single_element(value.get('a')),
+        'record': record,
+        'curated_relation': record is not None,
```

Comment:
Note on legacy, when an institution ID is actually referring to an existing institute, I am adding the corresponding record ID and setting curated to true, since otherwise it would be a raw string, not resolved.

In other situations (such as references or authors) the curated flag is really set only when a human actively curated the relation.